apps/dashboard/app/apps/ood_app.rb

```diff
@@ -73,10 +73,10 @@ def links
           new_tab: true
         )
       ].concat(
-        OodFilesApp.new.favorite_paths_with_name.map do |path, name|
+        OodFilesApp.new.favorite_paths.map do |path|
           OodAppLink.new(
-            nick_name: name,
-            title: path.to_s,
+            nick_name: path.instance_of?(FavoritePath) ? path.title : nil,
+            title: path.instance_of?(FavoritePath) ? path.uri.to_s : path.to_s,
```

Comment:
Instead of "nick_name" containing the main human readable title text like "Scratch" or "Project", introducing subtitle would be preferable.

Every link in the view still has its title, and you will optionally display the subtitle in small text, or on a separate line, etc. if it is present.

In this case, if just provided the path, the path is the title. If provided a path and a title string, the path is the subtitle.